units/src/fee_rate/mod.rs

```diff
@@ -201,15 +203,36 @@ impl FeeRate {
             None => None,
         }
     }
+
+    fn parse<S: AsRef<str>>(s: S) -> Result<Self, ParseError> {
+        let s = s.as_ref();
+        let (value, unit) = if let Some((value, unit)) = s.split_once(' ') {
+            (value, unit)
+        } else {
+            let pos = s
+                .find(|c: char| c.is_alphabetic())
+                .ok_or(ParseError(ParseErrorInner::MissingUnit))?;
+            (&s[..pos], &s[(pos + 1)..])
+        };
+        match unit {
+            "sat/kwu" => {
+                crate::parse::int_from_str(value)
+                    .map(Self::from_sat_per_kwu)
+                    .map_err(|error| ParseError(ParseErrorInner::InvalidSatPerKwu(error)))
+            },
+            "sat/vb" => Err(ParseError(ParseErrorInner::UnsupportedSatPerVB)),
```

Comment:
nit this should be `sat/vB` since it's virtual bytes not bits.

Reply:
Oh, yes, thanks!